test/integration/verify/controls/aws_lambdas.rb

```diff
@@ -7,6 +7,6 @@
   desc    "Ensure that our lambdas are correctly deployed"
 
   describe aws_lambdas() do
-    its ('count') { should > 1}
+    its('count') { should >= 1}
```

Comment:
For comparisons like this, the syntax is usually like `should be >= 1` - is is possible to use that here?
Would also suggest adding a space between `1` and `}` for consistency.

Reply:
Updated now.